assets/external_tools_meta.json

```diff
@@ -0,0 +1,56 @@
+{
+    "netmhc": {
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "132dc322da1e2043521138637dd31ebf",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHC-4.0/data.tar.gz",
+            "data_md5": "63c646fa0921d617576531396954d633",
+            "binary_name": "netMHC"
+        }
+    },
+    "netmhc_darwin": {
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "43519de644c67ee4c8ee04c673861568",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHC-4.0/data.tar.gz",
+            "data_md5": "63c646fa0921d617576531396954d633",
+            "binary_name": "netMHC"
+        }
+    },
+    "netmhcpan": {
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "94aa60f3dfd9752881c64878500e58f3",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCpan-4.0/data.Linux.tar.gz",
+            "data_md5": "26cbbd99a38f6692249442aeca48608f",
+            "binary_name": "netMHCpan"
+        }
+    },
+    "netmhcpan_darwin": {
+        "4.0": {
+            "version": 4.0,
+            "software_md5": "042d16b89adcf5656ca4deae06b55cc6",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCpan-4.0/data.Darwin.tar.gz",
+            "data_md5": "b4eef3c82852d677a98e0f7d753a36e2",
+            "binary_name": "netMHCpan"
+        }
+    },
+    "netmhcii": {
+        "2.2": {
+            "version": 2.2,
+            "software_md5": "918b7108a37599887b0725623d0974e6",
+            "data_url": "https://services.healthtech.dtu.dk/services/NetMHCII-2.2/data.tar.gz",
+            "data_md5": "11579b61d3bfe13311f7b42fc93b4dd8",
+            "binary_name": "netMHCII"
+        }
+    },
+    "netmhciipan": {
```

Comment:
Should not be in here right? Since it is not supported anymore 🤔

Reply:
Yes this still needs to be updated. This should be fixed with you're PR/changes then. Could you open a PR to `switch_to_epytope`?

Reply:
Yes! Still testing netmhc class 2 predictors because somehow I obtain empty results. Did not find the source of error yet